x-pack/legacy/plugins/infra/public/containers/logs/log_analysis/log_analysis_results.tsx

```diff
@@ -8,6 +8,23 @@ import createContainer from 'constate';
 import { useMemo, useEffect } from 'react';
 
 import { useLogEntryRate } from './log_entry_rate';
+import { GetLogEntryRateSuccessResponsePayload } from '../../../../common/http_api/log_analysis';
+
+type PartitionBucket = {
+  startTime: number;
+} & GetLogEntryRateSuccessResponsePayload['data']['histogramBuckets'][0]['partitions'][0];
+
+type PartitionRecord = Record<
+  string,
+  { buckets: PartitionBucket[]; topAnomalyScore: number; totalNumberOfLogEntries: number }
+>;
+
+export interface Results {
```

Comment:
`Results` => `LogAnalysisResults`?

If it's exported, I feel `Results` is too generic as a name and there's a higher chance it will collide with some other `Results` somewhere else

Reply:
Fair comment, I'll change this.